types/okta__okta-vue/index.d.ts

```diff
@@ -1,5 +1,5 @@
 // Type definitions for okta-vue 1.2
-// Project: https://github.com/okta/okta-oidc-js/tree/master/packages/okta-vue, https://github.com/okta/okta-oidc-js
+// Project: https://github.com/okta/okta-oidc-js
```

Comment:
Instead of removing broken okta-vue link entirely, update it to the new one - https://github.com/okta/okta-vue

This is, after all, a vue implementation, so that's really the most important link out of the two!

Reply:
:+1: Done. Thanks!